js/blocks/loader/edit.js

```diff
@@ -34,14 +33,22 @@ const formControls = ( props, block ) => {
 }
 
 const previewData = ( props, block ) => {
-	if ( typeof props.attributes.block_template !== 'undefined' && props.attributes.block_template.length > 0 ) {
-		return;
+	let params = new URLSearchParams();
+
+	for ( let attribute in props.attributes ) {
+
+		if ( ! props.attributes.hasOwnProperty( attribute ) ) continue;
+
+		if ( 'block_template' === attribute || 'block_preview' === attribute ) {
+			continue;
+		}
+
+		params.append( attribute, props.attributes[ attribute ] );
 	}
 
-	wp.apiFetch( { path: `/block-lab/v1/block-preview?slug=` + block.name } ).then(
+	wp.apiFetch( { path: '/block-lab/v1/block-preview?slug=' + block.name + '&' + params.toString() } ).then(
```

Comment:
Is there a better way of sending data along with the API fetch besides just appending it to the URL with `params.toString()`?